enzyme/BCLoad/ClangBCLoader.cpp

```diff
@@ -3,6 +3,8 @@
 
 #include "BCLoader.h"
 
+#include "BCLoaderNew.h"
+
```

Comment:
We will also need to register the new pass manager version to run automatically like the old pass manager version below.

Reply:
Need help . How do I register a module pass . Couldn't find in documentation about ModulePass. Then searched for in llvm code.

Reply:
https://medium.com/@mshockwave/writing-pass-instrument-for-llvm-newpm-f17c57d3369f

This blog should have some relevant information

Reply:
Sorry. Load shedding is going  on , stormy night, I am using  my phone . 
I will update these asap.

Reply:
> https://medium.com/@mshockwave/writing-pass-instrument-for-llvm-newpm-f17c57d3369f
> 
> This blog should have some relevant information

Thanks William.

Reply:
downgraded to llvm-11 . Now check-bcpass test pass. here is SS of /cblas_ddot_double.bc file not found error
![Screenshot_20220418_070659](https://user-images.githubusercontent.com/44522075/163739476-72498a3f-e925-49c0-82a1-36c24481d2b2.png)

Reply:
I think you need to pass -bcpath=/path/to/bclib

Reply:
Should I start writing the proposal already?

Reply:
Yes, most definitely

Reply:
> I think you need to pass -bcpath=/path/to/bclib

 'bcpath'  is an unknown argument to opt.

Reply:
> Yes, most definitely
20th april is deadline for proposal it's already 19 .
Will you be able to review my proposal before  submission and give me a feedback?

Reply:
manually setting the path to the bitcode file successfully loads the bitcode
![Screenshot_20220419_181427](https://user-images.githubusercontent.com/44522075/164001582-00614e3f-c9e2-44bf-afcd-afe7f450606b.png)